pub/index.html

```diff
@@ -14,11 +14,99 @@
 
         <link rel="stylesheet" href="css/normalize.css">
         <link rel="stylesheet" href="css/main.css">
+        <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
+        <link href='http://fonts.googleapis.com/css?family=Oswald|Open+Sans' rel='stylesheet' type='text/css'>
         <script src="js/vendor/modernizr-2.6.2.min.js"></script>
     </head>
     <body>
-
-
+        <div class="page">
+            <header>
+                <div class="top-container">
+                    <div class="search">
+                        <a href="">About</a>
+                        <a href="">Search</a>
+                        <a class="fa fa-search"></a>
+                    </div>
+                        <div class="logo">
+                            <img src="../pub/wave.png">
+                            <span class="hidden">SURF & PADDLE CO. BLOG</span>
+                        </div>
+                </div>    
+                <h1>SURF & PADDLE CO.</h1>
+                    <h2>An Ironyard Academy Project Winter 2015</h2>
+                    
+            </header>
+        <div class="main">
```

Comment:
Kinda nit-picky but the indentation is a little off here... Keeping those elements nested nicely will help you find bugs.